### PR TITLE
fixes #16592 - keep consistent for headline capitalization

### PR DESCRIPTION
**Pull request by junezhang**
Based on PatternFly Outlines Headline Style Capitalization, http://www.patternfly.org/styles/terminology-and-wording/#capitalization
We suggested to keep consistent in headline capitalization, capitalize the first letter of every word in headline.

As last PR #3841, I fixed the menu case, @ohadlevy suggested me to fix all the labels in one PR.
So I re-created one PR for updating all labels capitalization in Foreman.

Please have a review.
Thanks.

### CODE REVIEW

**Comment by junezhang** [comment]
@ohadlevy, @bbuckingham, @dLobatog, @tbrisker,  thanks for your comments on my last PR.
May I have your attention for reviewing my new PR, please, thanks!

File: app/views/subnets/index.html.erb [hunk 1]
```diff
@@ -7,7 +7,7 @@
   <thead>
     <tr>
       <th class="col-md-3"><%= sort :name, :as => s_("Subnet|Name") %></th>
-      <th class="col-md-2"><%= sort :network, :as => s_("Subnet|Network") %></th>
+      <th class="col-md-2"><%= sort :network, :as => s_("Subnet|Network Address") %></th>
       <th class="col-md-3"><%= _("Domains") %></th>
```
Comment (junezhang) [inline app/views/subnets/index.html.erb]: When reviewing the page, it is "Network address",  but in the coding it is "Network". so I added "Address" after Network.
If it is wrong, please correct me. Thanks.

**Comment by tbrisker** [comment]
@theforeman-bot ok to test

File: app/views/subnets/index.html.erb (hunk 1 above)
Comment (tbrisker) [reply app/views/subnets/index.html.erb]: For some reason this was changed to network address by the English translation file.

**Comment by tbrisker** [comment]
Sorry for the slow response, I was on a long vacation. OK to merge if all tests pass.

File: app/views/subnets/index.html.erb (hunk 1 above)
Comment (domcleal) [reply app/views/subnets/index.html.erb]: Don't create a second string (`Subnet|Network Address`) without removing the first (`Subnet|Network`), either change the existing translation or remove the original. (Not that attrs should be removed from the generated model attrs file.)

Ditto on every other change to strings that are already extracted via the model attrs file, they're duplicated.

Comment (tbrisker) [reply app/views/subnets/index.html.erb]: @domcleal I'm not quite sure what you mean, aren't the translation files automatically generated? what is the model attrs file?

Comment (domcleal) [reply app/views/subnets/index.html.erb]: Non-English translations are in Transifex, English translations are in this repo, and model attrs is locale/model_attributes.rb.

Comment (junezhang) [reply app/views/subnets/index.html.erb]: @domcleal what you meaning is if want to update the strings, fixing the capitalization problem, I need to update the English translation file: locale/en/foreman.po, Is that right?

**Comment by junezhang** [comment]
@domcleal I've updated the English translation PO file. Please help to review? 
When updating the PO file, I haven't found how to update these two items (Provisioning templates, Smart class parameters) in the navigation, so update them in the menu/loader.rb file.
Thanks.

File: app/views/subnets/index.html.erb (hunk 1 above)
Comment (junezhang) [reply app/views/subnets/index.html.erb]: @domcleal Update the English translation PO file, and re-submit a patch, please help to review. Thanks.

**Comment by ohadlevy** [comment]
@junezhang can you please rebase?
@tbrisker anything else this PR needs? thanks!

**Comment by junezhang** [comment]
test

**Comment by junezhang** [comment]
Rebase my commit, @tbrisker @domcleal Please have a review. Thanks.

**Comment by junezhang** [comment]
It's a long time for seeking feedback about my English translation PO file. so I think maybe it's not a right way. then I turn back to update the menu/loader.rb file and other .erb files. Please help to review it. Thanks.

**Comment by sean797** [comment]
Thanks a ton for doing this @junezhang 🥇 Looks good overall. If you have a chance to look at the test failures, can you? When they pass, it should be good to merge. If not I will raise a PR based off this one.

**Comment by junezhang** [comment]
Hi, @sean797, sorry for replying late. I've checked the test result, but have no idea about the test failure. Could you help on the test error, please? Thank you!

**Comment by sean797** [comment]
Sure, try this @junezhang 

```
$ git diff
diff --git a/test/integration_test_helper.rb b/test/integration_test_helper.rb
index 0c0e109..e1f7b3a 100644
--- a/test/integration_test_helper.rb
+++ b/test/integration_test_helper.rb
@@ -37,14 +37,14 @@ class ActionDispatch::IntegrationTest
   def assert_index_page(index_path,title_text,new_link_text = nil,has_search = true,has_pagination = true)
     visit index_path
     assert page.has_selector?('h1', :text => title_text), "#{title_text} was expected in the <h1> tag, but was not found"
-    (assert find_link(new_link_text).visible?, "#{new_link_text} is not visible") if new_link_text
+    (assert first(:link, new_link_text).visible?, "#{new_link_text} is not visible") if new_link_text
     (assert find_button('Search').visible?, "Search button is not visible") if has_search
     (assert has_content?("Displaying"), "Pagination 'Display ...' does not appear") if has_pagination
   end
 
   def assert_new_button(index_path,new_link_text,new_path)
     visit index_path
-    click_link new_link_text
+    first(:link, new_link_text).click
     assert_current_path new_path
   end

**Review by dLobatog — CHANGES_REQUESTED** [review]
Set to Waiting on contributor due to the failing test, thanks for the help @junezhang @sean797

**Comment by junezhang** [comment]
@sean797  Thanks for providing suggestion to fix the test failure. Currently I'm traveling in US. I will fix it next week. Sorry for any inconvenience.

**Comment by junezhang** [comment]
@sean797 I've submitted a commit, but it is still failed. So could you help to raise a PR based off this one, please?

**Comment by sean797** [comment]
@junezhang it works when I run them locally. Lets [test foreman] again, looks like it could be an intermittent failure

**Review by sean797 — APPROVED** [review]
\o/ tests passed, thanks @junezhang 👍

**Comment by junezhang** [comment]
@sean797 Thanks for your supporting. 
@dLobatog please have a review for the capitalization updating.

**Comment by ohadlevy** [comment]
@dLobatog bump?

**Review by dLobatog — APPROVED** [review]

**Comment by dLobatog** [comment]
Thank you @junezhang & @sean797

File: app/views/media/index.html.erb [hunk 2]
```diff
@@ -9,7 +9,7 @@
     <tr>
       <th class="col-md-2"><%= sort :name, :as => s_("Medium|Name") %></th>
       <th class="col-md-6"><%= sort :path, :as => s_("Medium|Path") %></th>
-      <th class="col-md-1"><%= sort :family, :as => s_("Medium|Os family") %></th>
+      <th class="col-md-1"><%= sort :family, :as => s_("Medium|Os Family") %></th>
```
Comment (dLobatog) [inline app/views/media/index.html.erb:12]: I think I may have missed this, should be `OS Family`.

File: app/views/ptables/index.html.erb [hunk 3]
```diff
@@ -7,8 +7,8 @@
   <thead>
     <tr>
       <th class="col-md-4"><%= sort :name, :as => s_("Ptable|Name") %></th>
-      <th class="col-md-1"><%= sort :family, :as => s_("Ptable|Os family") %></th>
-      <th class="col-md-4"><%= _("Operating systems") %></th>
+      <th class="col-md-1"><%= sort :family, :as => s_("Ptable|Os Family") %></th>
```
Comment (dLobatog) [inline app/views/ptables/index.html.erb:10]: Another one, OS family.

**Comment by sean797** [comment]
ah nice spot @dLobatog ! Created https://github.com/theforeman/foreman/pull/4700

**Comment by ares** [comment]
I set 1.16 in redmine issue and marked the linked PR to be included in 1.16 too.

**Pull request merged**